orttraining/orttraining/training_api/module.cc

```diff
@@ -148,8 +148,7 @@ Module::Module(const std::string& train_model_path_or_bytes,
                const Environment& env,
                const std::vector<std::shared_ptr<IExecutionProvider>>& providers,
                const std::optional<std::string>& eval_model_path_or_bytes)
-    : named_parameters_{named_parameters} {
-
+    : named_parameters_{named_parameters}, eval_user_input_count_(0) {
```

Comment:
I suggest putting it in the header file, as:
```
size_t eval_user_input_count_;
```
And do it for all primitive types such as integers/pointers.

Reply:
train_user_input_count_ should be initialized too.